README.md

```diff
@@ -1,2 +1,47 @@
 # comp130-journal
 Base repository for COMP130 research journal
+
+[] are used to denote the sourced paper with the number corresponding to the paper in the source list.
+
+## Section 1 - Introduction
+SDV(Self-Driving Vehicles) technology has come a long way since it's early beginnings in the 90's, with paper 4 documenting "challenges" that teams such as ARGO and Navlab demonstrating "thousands of kilometres of highway driving at speeds up to 130 kph with normal traffic" [4], and although this may sound very much like what current SDV's are capable of in retrospect, the technology used was all far too large and took up most of the room in the car that could be used to possibly seat passengers,  and the route the cars took were pre-planned with "highly accurate GPS waypoints"[4] so this sort of test doesn't really have any sort of practicality in the real world and was more used to show that the technology could work, along with that all other cars and vehicles were removed from the course, with this paper, it's easy to see how far SDV's have come since their rather primoridal beginnings. However, thanks however to the continued miniaturisation of technology and the use of new technologies such as ultra high resolution cameras that can help spot pedestrians up to 97 meters away [6], (which can far outstrip that of some road users), SDV's are now something that is available to the general public, thanks in no small part to Tesla, who in 2015 allowed it's users to utilise the experimental "Autopilot" mode [3], and as such many other companies such as Google and the car giant Toyota started to invest and develop in SDV technology, but this sudden development didn't come without it's critiques, as the normal person and academics alike started to ask, can you program a SDV to have acceptable ethics?, and who is at fault when the vehicle makes a mistake?
```

Comment:
You seem to write a lot of long run-on sentences -- this paragraph is only two sentences for example. This makes it difficult to read. Try to break your writing into shorter sentences, with one or two points per sentence.